Documentation/Contributing/PullRequests.md

```diff
@@ -0,0 +1,47 @@
+# Pull Requests
+
+## Prerequisites
+If you haven't contributed to a Microsoft project before, you may be asked to sign a [contribution license agreement](https://cla.microsoft.com/). 
+A comment in the PR will let you know if you do.
+
+> [!IMPORTANT]
+> If you are a Microsoft employee and are not a member of the [Microsoft organization on GitHub](https://github.com/Microsoft), please link your Microsoft and GitHub accounts on corpnet by visiting [Open Source at Microsoft](https://opensource.microsoft.com/) before you start your pull request. There's some process stuff you'll need to do ahead of time.
+
+## Creating a Pull Request
+When you are ready to submit a pull request, [create a pull request](https://github.com/microsoft/MixedRealityToolkit-Unity/compare/mrtk_development...mrtk_development?expand=1) targeting the [mrtk_development](https://github.com/microsoft/mixedrealitytoolkit-unity/tree/mrtk_development) branch.
+
+Read the guidelines and ensure your pull request meets the guidelines.
+
+* Make sure to reference any Issue / Feature Request or Task the PR relates to.
+* Check the pull request contains only files / changes related to the PR.
+* Check documentation is up to date and included. Check all public fields have comments.
+* If adding a new feature, check that tests are included to validate the feature (see [UnitTests](UnitTests.md)).
+* If fixing a bug, write a test to verify the bug fix.
+
+The project maintainers will review your changes. We aim to review all changes within three business days. Please address any review comments, push to your topic branch, and post a comment letting us know that there's new stuff to review.
+
+> [!NOTE]
+> All PR's submitted to the project will also be vetted according to the [MRTK coding standards guide](CodingGuidelines.md), so please review these before submitting your PR to ensure a smooth process.
+
+## Pull Request Guidelines
+These guidelines are based off of the [Google's engineering practices](https://google.github.io/eng-practices/review/developer/small-cls.html).
+
+### Keep pull requests small
+Smaller PRs are reviewed more quickly and thoroughly, are less likely to introduce bugs, easier to roll back, and easier to merge.
+
+Pull requests should be small enough that an engineer could review it in under 30 minutes. Try to make a minimal change that addresses just one thing. If you must create a large PR, split it into several PRs that go into either your local branch, or a feature branch of MRTK. Make sure any assets (e.g. fbx, obj files) are small.
```

Comment:
I'd actually say, in general, avoid adding media assets to MRTK (e.g fbx, png files) and instead re-use existing content. If it is necessary to add unique assets, make sure any assets are small